doc/source/examples/extended_examples/static_rom/static_rom_data_generation.py

```diff
@@ -0,0 +1,302 @@
+        symmetricalDim = True
+    else:
+        raise ValueError(f"Unsupported field dimensionality {field.component_count}")
+
+    settings = {
+        "pointsCoordinates": False,
+        "ids": compress_id_list(field.scoping.ids),
+        "location": "Nodal",
+        "unit": field.unit,
+        "unitDimension": {},
+        "unitFactor": 1.0,
+        "name": name,
+        "deformation": is_deformation,
+        "dimensionality": dimensionality,
+        "symmetricalDim": symmetricalDim,
+        "namedSelections": {},
+    }
+
+    with open(Path(path).joinpath("settings.json"), "w") as fw:
+        # Set default to convert Numpy int to int
+        json.dump(settings, fw, default=int, indent=4)
+
+
+def get_scoping(model: dpf.Model):
+    """Return scoping of unique node IDs connected to elements in model."""
+    op = dpf.operators.scoping.connectivity_ids(
+        mesh_scoping=model.metadata.meshed_region.elements.scoping,
+        mesh=model.metadata.meshed_region,
+        take_mid_nodes=True,
+    )
+    # Get output data
+    connected_nodes_scoping = op.outputs.mesh_scoping()
+    # Compress the list to only keep unique IDs
+    connected_nodes_scoping.ids = sorted(list(set(connected_nodes_scoping.ids)))
+    return connected_nodes_scoping
+
+
+def write_points(model: dpf.Model, scoping: dpf.Scoping, output_folder: str | Path):
+    """Write points.bin file."""
+    nodes = model.metadata.meshed_region.nodes
+    scoped_node_indices, _ = nodes.map_scoping(scoping)
+    points_coordinates = nodes.coordinates_field.data[scoped_node_indices]
+    write_binary(Path(output_folder).joinpath("points.bin"), points_coordinates)
+
+
+def write_doe_headers(output_folder: str | Path, name: str, parameters: dict):
+    """Write blank doe.csv file with headers."""
+    with open(Path(output_folder).joinpath("doe.csv"), "w", newline="") as fw:
+        writer = csv.writer(fw)
+        writer.writerow([name] + list(parameters.keys()))
+
+
+def write_doe_entry(output_folder: str | Path, snapshot_name: str, parameters: dict):
+    """Write entry to doe.csv file."""
+    with open(Path(output_folder).joinpath("doe.csv"), "a", newline="") as fw:
+        writer = csv.writer(fw)
+        writer.writerow([snapshot_name] + list(parameters.values()))
+
+
+def export_static_ROM_variation(
+    model: dpf.Model,
+    scoping: dpf.Scoping,
+    name: str,
+    output_folder: str | Path,
+    parameters: dict,
+    snap_idx: int = 0,
+    new_metadata: bool = False,
+):
+    """
+    Export static ROM data for one parameter variation.
+
+    Parameters
+    ----------
+    model : dpf.Model
+        DPF model with results data loaded.
+    scoping : dpf.Scoping
+        DPF nodal scoping for result export.
+    name : str
+        result quantity to export. Valid options are `displacement` and `stress`.
+    output_folder : str|Path
+        exported data will be stored in this folder. Use separate folders for each physics type.
```

Comment:
```suggestion
        Data to store exported data in. Use separate folders for each physics type.
```

Reply:
```suggestion
        Folder to store exported data in. Use separate folders for each physics type.
```
Changed 'Data' to 'Folder'